diff --git a/node_modules/@remix-run/react/dist/components.js b/node_modules/@remix-run/react/dist/components.js
index 4b170d9..808277f 100644
--- a/node_modules/@remix-run/react/dist/components.js
+++ b/node_modules/@remix-run/react/dist/components.js
@@ -719,7 +719,7 @@ function Meta() {
     let params = match.params;
     let routeModule = routeModules[routeId];
 
-    if (routeModule.meta) {
+    if (routeModule && routeModule.meta) {
       let routeMeta = typeof routeModule.meta === "function" ? routeModule.meta({
         data,
         parentsData,
diff --git a/node_modules/@remix-run/react/dist/links.js b/node_modules/@remix-run/react/dist/links.js
index a4890c5..60ed120 100644
--- a/node_modules/@remix-run/react/dist/links.js
+++ b/node_modules/@remix-run/react/dist/links.js
@@ -27,7 +27,7 @@ function getLinksForMatches(matches, routeModules, manifest) {
     var _module$links;
 
     let module = routeModules[match.route.id];
-    return ((_module$links = module.links) === null || _module$links === void 0 ? void 0 : _module$links.call(module)) || [];
+    return ((_module$links = (module ? module.links : undefined)) === null || _module$links === void 0 ? void 0 : _module$links.call(module)) || [];
   }).flat(1);
   let preloads = getCurrentPageModulePreloadHrefs(matches, manifest);
   return dedupe(descriptors, preloads);
